developers/building-monodevelop.md

```diff
@@ -104,10 +93,33 @@ Check out MD, configure using the Mac profile:
     make
     make run
 
+
 **NOTE:** DO NOT use *configure --select* on Mac - the Mac profile passes adds required arguments to the configuration of *main*.  Instead edit *profiles/mac* manually to add any additional modules.
 
 **IMPORTANT**: Do not install MonoDevelop into the install prefix. MonoDevelop will pick up libraries from the prefix automatically. If you configured with a non-default prefix, then when you build an app bundle, the **entire prefix** will be merged into the app bundle.
 
+**Things you might have to do before running the `configure` script**
+
+1. , tell aclocal where to find the pkgconfig M4 macro (pkg.m4), e.g.
+
+       export ACLOCAL_FLAGS="-I /Library/Frameworks/Mono.framework/Versions/Current/share/aclocal"
+
+   Comment: this did not appear necessary
+   
+2. Ensure that the Mono framework comes first in your PATH so you do not end up accidentally using conflicting versions of utilities from packages such as MacPorts:
+
+       export PATH="/Library/Frameworks/Mono.framework/Versions/Current/bin:$PATH"
+
+   Comment: this did not appear necessary
+
+3. Add the GTK+ libraries' location to the Mac dynamic loader path:
```

Comment:
This should be done via the stub launcher on mac. We automatically add that to the path.

Reply:
See https://github.com/mono/monodevelop/blob/master/main/build/MacOSX/monostub-utils.h#L219

Reply:
@Therzok OK, I removed this